documentation/docs/01-routing.md

```diff
@@ -318,6 +318,27 @@ A route can have multiple dynamic parameters, for example `src/routes/[category]
 
 > `src/routes/a/[...rest]/z.svelte` will match `/a/z` as well as `/a/b/z` and `/a/b/c/z` and so on. Make sure you check that the value of the rest parameter is valid.
 
+#### Validation
+
+A route like `src/routes/archive/[page]` would match `/archive/3`, but it would also match `/archive/potato`. We don't want that. You can ensure that route parameters are well-formed by adding _validators_ to your [`params`](/docs/configuration#files) directory...
```

Comment:
Perhaps this, rather than adding a comment to the code sample (which feels a bit duplicative with the type annotation):

```suggestion
A route like `src/routes/archive/[page]` would match `/archive/3`, but it would also match `/archive/potato`. We don't want that. You can ensure that route parameters are well-formed by adding a _validator_ — which takes the parameter string (`"3"` or `"potato"`) and returns `true` if it is valid — to your [`params`](/docs/configuration#files) directory...
```

Reply:
I thought about something like this, too, at first, but didn't find a concise way to write it out - I like the way you phrased it 👍